src/plugins/auto_batch/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "ov_auto_batch_plugin")
+
+file(GLOB SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+
+file(GLOB HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+
+ie_add_plugin(NAME ${TARGET_NAME}
+              DEVICE_NAME "BATCH"
+              SOURCES ${SOURCES} ${HEADERS}
+              VERSION_DEFINES_FOR auto_batch.cpp)
+
+target_link_libraries(${TARGET_NAME} PRIVATE ngraph inference_engine_transformations Threads::Threads)
+
+ie_add_api_validator_post_build_step(TARGET ${TARGET_NAME})
```

Comment:
Can we enable code style check for new plugin?

Reply:
great!